ln-visualizer/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: $APP_LNVISUALIZER_WEB_IP
+      APP_PORT: 80
+
+  web:
+    image: maxkotlan/ln-visualizer-web:v0.0.20@sha256:ab897a98a5b3640f8e26d423ef505537042a1ebd2804ef540d0ecbc88dc562c4
+    restart: on-failure
+    stop_grace_period: 1m
+    environment:
+      LN_VISUALIZER_API_URL: "http://${APP_LNVISUALIZER_API_IP}:5647"
+    networks:
+      default:
+        ipv4_address: "${APP_LNVISUALIZER_WEB_IP}"
+
+  api:
+    image: maxkotlan/ln-visualizer-api:v0.0.20@sha256:a0ab930436a3e223d4eee11114d3714b1bc21bb13fcb3c0280e2d8daca78719a
```

Comment:
The digest for this image should be: `sha256:cfea255f3343dbacf96c5470dcd442f1cc034337dfb46ecfbbc6d5c1978da1a4`

Reply:
Changed to `sha256:cfea255f3343dbacf96c5470dcd442f1cc034337dfb46ecfbbc6d5c1978da1a4`